File: C#/백준/Silver/10190. Acronyms/Acronyms.cs
using System;

namespace q{
    class w{
        static void Main(){
            int a = int.Parse(Console.ReadLine());
            for (int i=0;i<a;i++){
                String[] aa = Console.ReadLine().Split();
                String s1 = aa[0];
                int n = int.Parse(aa[1]);
                Console.WriteLine(s1);
                for (int j=0;j<n;j++){
                    String b = Console.ReadLine();
                    String[] bb =b.Split();
                    String s2 = "";
                    for (int k=0;k<bb.Length;k++){
                        s2+=bb[k][0];
                    }
                    if(s1.Equals(s2)){
                        Console.WriteLine(b);
                    }
                }
            }
        }
    }
}

File: Java/백준/Silver/10190. Acronyms/Acronyms.java
import java.util.Scanner;

public class Main{
    public static void main(String[] args){
        Scanner in = new Scanner(System.in);
        int a = in.nextInt();
        for (int i=0;i<a;i++){
            String s1 = in.next();
            int b = in.nextInt();
            in.nextLine();
            System.out.println(s1);
            for (int j=0;j<b;j++){
                String s = in.nextLine();
                String[] aa = s.split(" ");
                String s2 = "";
                for (int k=0;k<aa.length;k++){
                    s2+=aa[k].charAt(0);
                }
                if (s1.equals(s2)){
                    System.out.println(s);
                }
            }
        }
    }
}